src/nvim/misc1.c

```diff
@@ -1281,11 +1280,13 @@ int plines_win_nofold(win_T *wp, linenr_T lnum)
    * Add column offset for 'number', 'relativenumber' and 'foldcolumn'.
    */
   width = wp->w_width - win_col_off(wp);
-  if (width <= 0)
+  if (width <= 0) {
     return 32000;
```

Comment:
I guess it's fine, but let's add a comment ("bigger than the number of lines of the screen")